src/main/java/com/synopsys/integration/alert/workflow/startup/component/ConfigurationLogger.java

```diff
@@ -35,27 +35,27 @@
 import com.synopsys.integration.alert.common.rest.ProxyManager;
 
 @Component
-@Order(4)
+@Order(5)
```

Comment:
Perhaps we should make our standard steps increments of 10? That way we can fit temporary/new steps in between existing without re-ordering everything every time

Reply:
I will implement that change in 5.3.0 as an enhancement when this change is merged.